spec/views/layout/application.html.erb_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe 'navigation bar', type: :view do
+
+  shared_examples 'a registered User' do
+    it 'should link to home' do
+      expect(page).to have_link('Wimi Portal', href: current_url)
+    end
+
+    it 'should link to profile' do
+      expect(page).to have_link('Profile', href: user_path(@user))
+    end
+
+    it 'should link to logout' do
+      expect(page).to have_link('Logout', href: destroy_user_session_path)
+    end
+  end
+
+  context 'for a superadmin' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @user = FactoryGirl.create(:user, superadmin: true)
+      login_as @user
+      visit root_path
+    end
+
+    it 'should show 3 links' do
+      expect(page).to have_css('nav a', count: 3)
+    end
+  end
+
+  context 'for a wimi' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @chair = FactoryGirl.create(:chair)
+      @wimi_user = FactoryGirl.create(:user)
+      @user = FactoryGirl.create(:wimi, user: @wimi_user, chair: @chair).user
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should show 4 links' do
+      expect(page).to have_css('nav a', count: 4)
+    end
+  end
+
+  context 'for a hiwi' do
+    it_behaves_like 'a registered User'
+    before(:each) do
+      @user = FactoryGirl.create(:user)
+      @chair = FactoryGirl.create(:chair)
+      @project = FactoryGirl.create(:project, chair: @chair, status: true)
+      @user.projects << @project
+      login_as @user
+      visit root_path
+    end
+
+    it 'should link to projects' do
+      expect(page).to have_link('Projects', href: projects_path)
+    end
+
+    it 'should show 4 links' do
+      expect(page).to have_css('nav a', count: 4)
```

Comment:
Also: Ich teste genau das und alles, was er sehen soll. Konkret heißt das zB: (Superadmin)
- Er verhält sich wie ein registrierter Nutzer, heißt:
-- Er sieht einen Link zur Startseite
-- Er sieht einen Link zu seinem Profil
-- Er sieht einen Link zum Logout
- Außerdem sieht er insgesamt nur 3 Links, eben diese spezifizierten

Damit habe ich alle Fälle getestet, die durch meine User-Stories spezifiziert waren. Es ist kein Fall möglich, in dem alle Tests grün sind, aber die Story nicht mehr geht. Und wenn ich das richtig verstehe, ist doch genau das die Anforderung?
@swt-r0ckz : Es ist so, dass der Superadmin keine weiteren Links sieht, die ein registrierter Nutzer sehen kann. Von daher, doch.

Letztlich folgendes: Ich freue mich natürlich über Kommentare über meinen PR, aber es hilft mir echt überhaupt nicht weiter, wen die Aussage (im Kern) nur ist: "Das ist nicht gut! Mach das anders!", aber keine Idee geboten wird, WIE ich das ändern soll. Wenn ihr da genaue Vorstellungen habt, dann schreibt sie doch bitte gleich rein.

Trotz allem: Danke für die Anmerkungen.